domain/src/test/kotlin/no/nav/su/se/bakover/domain/søknadsbehandling/StatusovergangTest.kt

```diff
@@ -557,7 +666,18 @@ internal class StatusovergangTest {
                 nySaksbehandler = saksbehandler,
             ).getOrFail() shouldBe beregnetInnvilget
                 .medFritekstTilBrev(underkjentInnvilget.fritekstTilBrev)
-                .copy(attesteringer = Attesteringshistorikk.create(listOf(underkjentInnvilget.attesteringer.hentSisteAttestering())))
+                .copy(
+                    attesteringer = Attesteringshistorikk.create(listOf(underkjentInnvilget.attesteringer.hentSisteAttestering())),
+                    søknadsbehandlingsHistorikk = beregnetInnvilget.søknadsbehandlingsHistorikk.leggTilNyeHendelser(
+                        listOf(
+                            nySøknadsbehandlingshendelse(handling = SøknadsbehandlingsHandling.Simulert),
+                            nySøknadsbehandlingshendelse(handling = SøknadsbehandlingsHandling.SendtTilAttestering),
```

Comment:
Jeg ser tanken med å finne et felles ord for saksbehandler og attestant, og ha en liste med begge sine handlinger. Men det kan vi tenke på til siden.